sonar-update-center-common/src/main/java/org/sonar/updatecenter/common/Version.java

```diff
@@ -145,16 +151,19 @@ public String toString() {
     return name;
   }
 
-  public static Version create(String version) {
-    return new Version(version);
-  }
+  public static Version create(String versionStr ) { return new Version(versionStr, versionStr); }
+
+  public static Version create(String versionStr, String fromString ) { return new Version(versionStr, fromString); }
+
+  public static Version create(Version version, String fromString ) { return new Version(version.name, fromString); }
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this line start at column 5. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AIndentationCheck)